File: src/main/java/pl/patrykdolata/lifemanager/LifeManagerApplication.java
package pl.patrykdolata.lifemanager;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import pl.patrykdolata.lifemanager.config.SecurityProperties;

@SpringBootApplication
@EnableConfigurationProperties({SecurityProperties.class})
public class LifeManagerApplication {

	public static void main(String[] args) {
		SpringApplication.run(LifeManagerApplication.class, args);
	}

}

File: src/main/kotlin/pl/patrykdolata/lifemanager/LifeManagerApplication.kt
package pl.patrykdolata.lifemanager

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import pl.patrykdolata.lifemanager.config.SecurityProperties
import pl.patrykdolata.lifemanager.mapper.UserMapper
import pl.patrykdolata.lifemanager.model.NewUser

@SpringBootApplication
@EnableConfigurationProperties(SecurityProperties::class)
class LifeManagerApplication

fun main(args: Array<String>) {
	runApplication<LifeManagerApplication>(*args)
}
